File: JavaScript/8js 1/derive.js
/*
|--------------------------------------------------------------------------
| Take the Derivative
|--------------------------------------------------------------------------
|
*/

function derive(coefficient, exponent) {
   return `${coefficient * exponent}x^${exponent - 1}`;
}


const derive = (coefficient, exponent) => `${coefficient * exponent}x^${exponent - 1}`;


function derive(c, e) {
   return c * e + "x^" + (e - 1);
}


console.log(derive(7, 8), "56x^7");
console.log(derive(5, 9), "45x^8");

File: Python/8py 1/derive.py
"""
|--------------------------------------------------------------------------
| Take the Derivative
|--------------------------------------------------------------------------
|
"""

def derive(coefficient, exponent): 
   return f"{coefficient*exponent}x^{exponent-1}"


derive = lambda c,e: f"{c*e}x^{e-1}"


def derive(coefficient, exponent): 
   return("{}x^{}".format(coefficient*exponent, exponent-1))


print(derive(7,8), "56x^7")
print(derive(5,9), "45x^8")